src/styles/global.css

```
@@ -13,4 +13,79 @@
+.title-bar-text {
+  cursor: default;
+  user-select: none;
+}
+
+.icon-selected {
+  position: absolute;
+  width: 36px;
+  height: 36px;
+  margin: -2px;
+
+  z-index: 1;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)